plugins/outputs/newrelic/newrelic.go

```diff
@@ -15,8 +17,9 @@ import (
 type NewRelic struct {
 	harvestor    *telemetry.Harvester
 	dc           *cumulative.DeltaCalculator
-	InsightsKey  string `toml:"insights_key"`
-	MetricPrefix string `toml:"metric_prefix"`
+	InsightsKey  string            `toml:"insights_key"`
```

Comment:
@hsinghkalsi Were these whitespace changes intended?

Reply:
@JimHagan yes we added new configurable parameter for timeout